src/api.rs

```diff
@@ -18,12 +18,15 @@
 //  along with DNS updater.  If not, see <https://www.gnu.org/licenses/>.
 //
 use log::{debug, trace};
-use reqwest::{Client, Method, RequestBuilder};
+use reqwest::{Client, ClientBuilder, Method, RequestBuilder};
 use serde::{Deserialize, Serialize};
-use std::ops::Deref;
 use std::time::Duration;
+use std::{env, ops::Deref};
 use strum_macros::{Display, EnumString};
 
+/// Environment variable for name.com API key.
+const ENV_NAMECOM_REQUEST_PROXY: &'static str = "NAMECOM_REQUEST_PROXY";
```

Comment:
Would it make sense to use some of the well-known env vars like `HTTP_PROXY` `HTTPS_PROXY`?